### DISCUSSION TITLE
Add "See on TurboWarp" link on project page

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#2656] [Ideas]
Many users now add a turbowarp link on their instructions section that you have to select or to copy and paste. Many don't. It would be cool to automatically show a "See on TurboWarp" link that you can directly click.

**Comment by Shluffy** [comment]
paste this in the console and it will redirect you to the project on turbowarp:
"window.location.href = `https://turbowarp.org/${window.location.href.substring( 33)}`;" (qoute for exact code)
why not make the addon yourself?

**Comment by retronbv** [comment]
I have a ***messy*** userscript [(here)](https://github.com/retronbv/userscripts/blob/master/TurbowarpScratchMod.user.js) that does this.

**Comment by nexensys** [comment]
Idea: If there is a link in the desc. and it has addons enabled, make the button enable the same addons? (kinda bad wording, sorry.)
